lib/gitlab_core.ml

```diff
@@ -1478,6 +1486,21 @@ struct
         API.post ~token ~uri ~body ~expected_code:`Created (fun s ->
             Lwt.return (Gitlab_j.project_access_token_of_string s))
     end
+
+    module Issue = struct
+      let list ?token ~project_id () =
```

Comment:
What do you think about adding filters here as in the `merge_requests` function?

Reply:
Great idea, there's already some examples of how to do that.